src/SOS/SOS.UnitTests/Debuggees/Directory.Build.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <Import Project="$(MSBuildThisFileDirectory)..\..\..\Directory.Build.props"/>
-  
+  <Import Project="$(RepoRoot)/eng/AuxMsbuildFiles/SdkPackOverrides.targets"/>
```

Comment:
Does this prop file get copied?  Isn't it used for just the prebuilt debuggees? If so, does it need to include the SdkPackOverrides?

Reply:
It's a build time artifact so no file copy. If you don't fix it like this, we'll end up testing torn runtimes (single file and the other test against one runtime).